source/client/process_impl.cc

```diff
@@ -531,21 +446,21 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const std::vector<UriP
             : Http1PoolImpl::ConnectionReuseStrategy::MRU);
     cluster_manager_factory_->setPrefetchConnections(options_.prefetchConnections());
     if (tracing_uri != nullptr) {
-      setupTracingImplementation(bootstrap, *tracing_uri);
-      addTracingCluster(bootstrap, *tracing_uri);
+      setupTracingImplementation(*bootstrap, *tracing_uri);
```

Comment:
While I would naturally lean towards this, our internal style requires a single variable name for StatusOr objects, so I have been muscle memory trained.

I don't think this is in the public style guide, so if you believe this will be better with two variable names for readability, let me know. Dereferencing a StatusOr is different than dereferencing a pointer, it essentially just calls .value() on it.

Going to resolve for now as this was "intended", but do reopen if you think the other approach is more readable.